README.md

```diff
@@ -43,6 +46,9 @@ A list of companies that have paid Developer Community Writer Programs.
 - [CircleCI](https://circleci.com/blog/guest-writer-program/)  - Up to $300 per piece
   > Technical tutorials with code. Pick from a list of possible articles.
 
+- [Civo](https://www.civo.com/write-for-us) - $300- $500 per article
```

Comment:
Make the end of this line `$300-$500 per piece` to be consistent with others in the list.